libraries/AP_OpticalFlow/OpticalFlow.cpp

```diff
@@ -75,7 +75,7 @@ OpticalFlow::OpticalFlow(AP_AHRS_NavEKF &ahrs)
 {
     AP_Param::setup_object_defaults(this, var_info);
 
-    memset(&_state, 0, sizeof(_state));
+    memset((void *)&_state, 0, sizeof(_state));
```

Comment:
Or, you know, just nuke the thing entirely.  If you ever allocate one of these on the stack you're doing something wrong.  You can also remove the line below it.